.circleci/config.yml

```diff
@@ -57,13 +71,55 @@ jobs:
               echo "Failure: Artifacts Not Present"
               exit 1
             fi
+      - persist_to_workspace:
+          root: .
+          paths:
+            - artifacts/*
 
       - store_artifacts:
           path: "build-wasm"
           destination: "wasm-without-wormhole"
+  publish_to_github:
+     docker:
+       - image: cibuilds/github:0.10
+     steps:
+       - attach_workspace:
+          # Must be absolute path or relative path from working_directory
+          at: ./
+       - run:
+          name: "Publish Release on GitHub"
+          command: |
+
+            export COMMIT=$(echo $CIRCLE_SHA1 | cut -c -7)
+            export VERSION=$(cat ./artifacts/BERGAMOT_VERSION)
+            VERSION=$VERSION+$COMMIT
+            ls -lsa ./artifacts/ > ./artifacts/FILESIZES
+            cat ./artifacts/SHA256-1 ./artifacts/SHA256-2 > ./artifacts/SHA256
+            rm ./artifacts/SHA256-1
+            rm ./artifacts/SHA256-2
+            rm ./artifacts/BERGAMOT_VERSION
+
+            ghr -t ${GHTOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} -delete ${VERSION} ./artifacts/
 
 workflows:
   build:
       jobs:
-          - build-with-wormhole
-          - build-without-wormhole
\ No newline at end of file
+          - build-with-wormhole:
+              filters:
+                tags:
+                  only: /^v.*/
+          - build-without-wormhole:
+              filters:
+                tags:
+                  only: /^v.*/
```

Comment:
A question. Does these steps mean "build-with-wormhole" and "build-without-wormhole" jobs will run only when a new tag is created? We want this behavior only for "publish_to_github" job. Right?

Reply:
Never mind. This is correct.